nixos/modules/services/misc/gitlab.nix

```diff
@@ -475,6 +526,7 @@ in {
         gitAndTools.git
         openssh
         nodejs
+        procps
```

Comment:
I don't see q requirement for this in the installation documents, the requirements or the changelog.